### ISSUE TITLE
Property subscriptionStatus does not exist on type Current.User

### ISSUE THREAD

**Issue by adriatic** [generalui/redwood-stripe#25]
Not sure whether this is related to the problem I discovered and reported to Redwood team (TypeScript broken) or not. Does this problem tell you anything?

<img width="737" alt="image" src="https://user-images.githubusercontent.com/2712405/182058606-c52ac88d-c347-4270-8c31-97cc79182d55.png">

I checked the code in the original repo (this one) and the line with `currentUser.subscriptionStatus` appears only once - in the above screenshot. Upon further inspection the current user is a part of the redwood code

`import { getCurrentUser } from '../../../api/src/lib/auth'`

**Comment by standup75** [comment]
This works for me, I see you seem to have some IDE config issues. You can try option+command+shift+R to reload ts server.
Otherwise make sure that you have the recommended extensions installed.
```
    "dbaeumer.vscode-eslint",
    "eamodio.gitlens",
    "ofhumanbondage.react-proptypes-intellisense",
    "mgmcdermott.vscode-language-babel",
    "wix.vscode-import-cost",
    "pflannery.vscode-versionlens",
    "editorconfig.editorconfig",
    "prisma.prisma",
    "graphql.vscode-graphql",
    "csstools.postcss",
    "bradlc.vscode-tailwindcss"
```

**Comment by standup75** [comment]
I close because unrelated to this project

**Comment by adriatic** [comment]
I have the same VSCode extensions. let's take this particular issue off the table. As I am not sure whether I made that screenshot from my own code for Redwood Tutorial TypeScript - code that I typed in myself to be able to verify the problems I am having with redwood-stripe app, let's leep this issue closed as you did.

If I run in redwood-stripe app TypeScript problems after checking all your today's response to my last set of issues, I will create a new redwood-stripe TypeScript problem (I am sure that I will have to, as Typescript generated file `graphql.d.ts file is bad`) Will deal with that by the end of my day
